### DISCUSSION TITLE
JSON for watering

### DISCUSSION THREAD

**Discussion by kolyanks** [RavenSystem/esp-homekit-devices#1635] [MEPLHAA Scripts]
Sonoff 4CH Irrigation System Automation does not work, what to do

**Comment by mario-website** [comment]
Quick answer is:
change "t": 20 [Water Value](https://github.com/RavenSystem/esp-homekit-devices/wiki/Water-Valve) to "t": 1 [Switch](https://github.com/RavenSystem/esp-homekit-devices/wiki/Switches) 

Full explanation:
Hi, I wanted to open a separate topic about it but I’m glad that I’m not alone in this world who have the same issue :) It looks like there is a problem with HomeKit itself! 
I started interesting about automation about 2 months ago. I purchased Gardena system (paid 350 euro for full set) and have the same issue! I even send to gardena (husqvarna) email with screen shoots over a month ago where I explained issue - no respond till now.
I started to looking for another solution and found that Sonoff 4CH can do this too. It wasn’t easy to learn how to flash but finally manage to do it. Then I saw that the same issue exist like in gardena so I tried to change "Valve Type” to any other - no change. So I changed also "Service Type” to „Switch” and now in automation all valves appears!

**Comment by mario-website** [comment]
@kolyanks, would you take a look to [THIS POST](https://github.com/RavenSystem/esp-homekit-devices/discussions/1636)?
